app/src/main/java/com/framgia/soundclound/data/source/remote/LoadAsync.java

```diff
@@ -41,12 +42,12 @@ protected String doInBackground(String... strings) {
             connection.setRequestMethod("GET");
             int responseCode = connection.getResponseCode();
             if (responseCode != HttpURLConnection.HTTP_OK) {
-                throw new IOException("HTTP Code" + responseCode);
+                new IOException("HTTP Code" + responseCode);
```

Comment:
ERROR: [CORRECTNESS][RV_EXCEPTION_NOT_THROWN] new java.io.IOException(String) not thrown in com.framgia.soundclound.data.source.remote.LoadAsync.doInBackground(String[])
http://findbugs.sourceforge.net/bugDescriptions.html#RV_EXCEPTION_NOT_THROWN